lib/block.ml

```diff
@@ -119,7 +119,7 @@ let pp_lines syntax t =
   let pp =
     match syntax with
     | Some Syntax.Cram -> Fmt.fmt "  %s"
-    | Some Syntax.Mli ->
+    | Some Syntax.Mli | Some Mld ->
```

Comment:
The `Syntax.` prefix is repeated because it's a new feature of OCaml that wasn't always available. I think the better style now that Mdx requires 4.08 would be to write the prefix once, then leave it out in every other cases.

Reply:
Happy to go with whatever style the maintainers want :-)

Reply:
I'm not particularly a fan of "prefix once", since it makes the matches look inconsistent (even worse in case of resorting the match branches). The syntactic support probably has been around for ages, I didn't know it wasn't supported earlier.